src/vs/workbench/api/common/extHostWorkspace.ts

```diff
@@ -187,6 +188,8 @@ export class ExtHostWorkspace implements ExtHostWorkspaceShape, IExtHostWorkspac
 
 	private _trusted: boolean = false;
 
+	private readonly _EditSessionIdentityProviders: { [scheme: string]: vscode.EditSessionIdentityProvider } = Object.create(null);
```

Comment:
* use `_<lowerCase>`-naming
* use `Map<string,vscode.EditSessionIdentityProvider>` not an object literal